.github/workflows/netlify.yml

```diff
@@ -0,0 +1,83 @@
+name: "📤 Upload Preview Build to Netlify"
+on:
+  workflow_run:
+    workflows: ["🌐 Build the matrix.org website"]
+    types:
+      - completed
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      # There's a 'download artifact' action but it hasn't been updated for the
+      # workflow_run action (https://github.com/actions/download-artifact/issues/60)
+      # so instead we get this mess:
+      - name: "📥 Download artifact"
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var artifacts = await github.actions.listWorkflowRunArtifacts({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: ${{github.event.workflow_run.id }},
+              });
+              var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
+                return artifact.name == "merged-content-artifact"
+                })[0];
+                var download = await github.actions.downloadArtifact({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  artifact_id: matchArtifact.id,
+                  archive_format: 'zip',
+                });
+                var fs = require('fs');
+                fs.writeFileSync('${{github.workspace}}/content.zip', Buffer.from(download.data));
+
+                var prInfoArtifact = artifacts.data.artifacts.filter((artifact) => {
+                  return artifact.name == "pr.json"
+                })[0];
+                var download = await github.actions.downloadArtifact({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  artifact_id: prInfoArtifact.id,
+                  archive_format: 'zip',
+                });
+                var fs = require('fs');
+                fs.writeFileSync('${{github.workspace}}/pr.json.zip', Buffer.from(download.data));
```

Comment:
if we really need this stuff.. maybe wee should write our own action for it.

Reply:
Probably, although the ideal scenario really would be for [the download-artifact action](https://github.com/actions/download-artifact) to be fixed, as pointed out by the comment. This comes from [matrix-react-sdk](https://github.com/matrix-org/matrix-react-sdk/blob/develop/.github/workflows/netlify.yaml) BTW.

Can we consider this out of scope for this PR though?

Reply:
> Probably, although the ideal scenario really would be for [the download-artifact action](https://github.com/actions/download-artifact) to be fixed, as pointed out by the comment. 

Sure, but given the issue has been open a year with no reply from the maintainers, let's not hold our breath on that one.

> This comes from [matrix-react-sdk](https://github.com/matrix-org/matrix-react-sdk/blob/develop/.github/workflows/netlify.yaml) BTW.

ok, so this is at least our second copy of it!

> Can we consider this out of scope for this PR though?

I guess so. I do think we should do something else before we make a third copy.

My understanding is that once you've got the javascript, writing an action is pretty easy.